### PR TITLE
tldr: update Korean translation, common/t*: add Korean translation

### PR DESCRIPTION
**Pull request by CodePsy-2001**
- **common/t*: add Korean translation**
- **tldr: update Korean translation**

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by CodePsy-2001** [comment]
I ran a local pull request test, but it's hard to interpret the log. What is the problem with the translation file?

File: pages.ko/common/tar.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+- 파일 확장자를 사용하여 압축 프로그램을 [자]동으로 결정하고 압축 아카이브 [생]성 후 [파]일로 저장:
+
+`tar caf {{경로/대상/파일.tar.xz}} {{경로/대상/파일1 경로/대상/파일2 ...}}`
+
+- tar [파]일의 내용을 [상세히] [목]록화:
+
+`tar tvf {{경로/대상/원본.tar}}`
+
+- 아카이브 [파]일에서 패턴과 일치하는 파일 [추]출:
```
Comment (Zamoca42) [inline pages.ko/common/tar.md:35]:
```suggestion

> 아카이브 유틸리티.
> 종종 `gzip` 또는 `bzip2`와 같은 압축 방법과 결합하여 사용됩니다.
> 더 많은 정보: <https://www.gnu.org/software/tar>.

- 아카이브 생성 후 파일로 저장:

`tar cf {{경로/대상/파일.tar}} {{경로/대상/파일1 경로/대상/파일2 ...}}`

- g[z]ipped 아카이브 생성 후 파일로 저장:

`tar czf {{경로/대상/파일.tar.gz}} {{경로/대상/파일1 경로/대상/파일2 ...}}`

- 디렉토리에서 상대 경로를 사용하여 g[z]ipped (압축된) 아카이브 생성:

`tar czf {{경로/대상/파일.tar.gz}} --directory={{경로/대상/폴더}} .`

- (압축된) 아카이브 파일을 현재 디렉토리에 상세히 추출:

`tar xvf {{경로/대상/원본.tar[.gz|.bz2|.xz]}}`

- (압축된) 아카이브 파일을 대상 디렉토리에 추출:

`tar xf {{경로/대상/원본.tar[.gz|.bz2|.xz]}} --directory={{경로/대상/폴더}}`

- 파일 확장자를 사용하여 압축 프로그램을 자동으로 결정하고 압축 아카이브 생성 후 파일로 저장:

`tar caf {{경로/대상/파일.tar.xz}} {{경로/대상/파일1 경로/대상/파일2 ...}}`

- tar 파일의 내용을 상세히 목록화:

`tar tvf {{경로/대상/원본.tar}}`

- 아카이브 파일에서 패턴과 일치하는 파일 추출:
```

File: pages.ko/common/tarsnap.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# tarsnap
+
+> 원격 Tarsnap 암호화 백업을 조작.
+> 참고: `/usr/local/etc/tarsnap.conf` 또는 `~/.tarsnaprc`에서 설정하면 키 파일과 캐시 디렉토리를 지정할 필요가 없습니다.
+> 같이 보기: `tarsnap-keygen`.
+> 더 많은 정보: <https://www.tarsnap.com/man-tarsnap.1.html>.
+
+- 하나 이상의 파일 또는 디렉토리에 대한 백업 아카이브 [생성], 암호화 키와 캐시 디렉토리 지정:
```
Comment (Zamoca42) [inline pages.ko/common/tarsnap.md:8]:
```suggestion
- 하나 이상의 파일 또는 디렉토리에 대한 백업 아카이브 생성, 암호화 키와 캐시 디렉토리 지정:
```

File: pages.ko/common/tarsnap.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+- 저장된 아카이브 목록 표시:
+
+`tarsnap --list-archives --keyfile {{경로/대상/키_파일}}`
+
+- 특정 아카이브 [삭제]:
+
+`tarsnap -d --keyfile {{경로/대상/키_파일}} --cachedir {{경로/대상/캐시_디렉토리}} -f {{아카이브_이름}}`
+
+- 특정 아카이브의 내용을 [상세] 모드로 목록화:
```
Comment (Zamoca42) [inline pages.ko/common/tarsnap.md:24]:
```suggestion
- 특정 아카이브 삭제:

`tarsnap -d --keyfile {{경로/대상/키_파일}} --cachedir {{경로/대상/캐시_디렉토리}} -f {{아카이브_이름}}`

- 특정 아카이브의 내용을 상세 모드로 목록화:
```

File: pages.ko/common/transfersh.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`transfersh {{경로/대상/파일}}`
+
+- 진행 표시줄을 보여주며 파일 업로드 (Python 패키지 `requests_toolbelt` 필요):
+
+`transfersh --progress {{경로/대상/파일}}`
+
+- 다른 파일 이름으로 파일 업로드:
+
+`transfersh --name {{파일이름}} {{경로/대상/파일}}`
```
Comment (Zamoca42) [inline pages.ko/common/transfersh.md:16]:
```suggestion
`transfersh --name {{파일명}} {{경로/대상/파일}}`
```

File: pages.ko/common/tuir.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+`/{{서브레디시트_이름}}`
+
+- 링크 열기:
+
+`o`
+
+- 시작 시 특정 서브레디시트 열기:
+
+`tuir -s {{서브레디시트_이름}}`
```
Comment (Zamoca42) [inline pages.ko/common/tuir.md:21]:
```suggestion
- 서브레딧 열기:

`/{{서브레딧_이름}}`

- 링크 열기:

`o`

- 시작 시 특정 서브레딧 열기:

`tuir -s {{서브레딧_이름}}`
```

File: pages.ko/common/twine.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+- PyPI에 업로드:
+
+`twine upload dist/*`
+
+- Test PyPI [저]장소에 업로드하여 검증:
+
+`twine upload -r testpypi dist/*`
+
+- 지정된 [사]용자 이름과 [비]밀번호로 PyPI에 업로드:
```
Comment (Zamoca42) [inline pages.ko/common/twine.md:14]:
```suggestion
- Test PyPI 저장소에 업로드하여 검증:

`twine upload -r testpypi dist/*`

- 지정된 사용자 이름과 비밀번호로 PyPI에 업로드:
```

File: pages.ko/common/twine.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+`twine upload -u {{사용자_이름}} -p {{비밀번호}} dist/*`
```
Comment (Zamoca42) [inline pages.ko/common/twine.md:16]:
```suggestion
`twine upload -u {{사용자명}} -p {{비밀번호}} dist/*`
```

**Review by Zamoca42 — CHANGES_REQUESTED** [review]

**Review by Zamoca42 — APPROVED** [review]
